blockchain/fullblocktests/generate.go

```diff
@@ -2546,6 +2736,77 @@ func Generate(includeLargeReorg bool) (tests [][]TestInstance, err error) {
 	g.AssertTipNumRevocations(1)
 	accepted()
 
+	// Create block that has a revocation with more payees than expected.
+	//
+	//   ... -> brt1(24) -> brt4(25)
```

Comment:
Remember, blocks that are intended to be rejected should be depicted as a side chain because they can't possibly be the main chain as this depicts.

```Go
//   ... -> brt1(24)
//                  \-> brt4(25)
```

This applies to all the other rejected blocks below too.